### PR TITLE
[Fix] Storage Retention and PVC prometheus

### PR DESCRIPTION
**Pull request by gabrielifg**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

fixing

* **What is the current behavior?** (You can also link to an open issue here)

There is no creation of the PVC referring to prometheus. That is, the data is not being persisted.

* **What is the new behavior (if this is a feature change)?**

Now there is the creation of the PVC in addition to the TSDB persistent has to go up from 80h to 15d.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

dojot/dojot#2270

* **Other information**:

### CODE REVIEW

**Review by danielapetito — APPROVED** [review]

**Review by ewerton-silva00 — APPROVED** [review]
Great. Thank you!

File: roles/volumes/storageclass/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- name: Create StorageClass
+  k8s:
+    kubeconfig: "{{ dojot_kubeconfig_file_path | default(omit) }}"
+    state: present
+    definition: "{{ item }}"
+  register: result
+  until: result.failed == 0
+  retries: 3
```
Comment (marianoleonardo) [inline roles/volumes/storageclass/tasks/main.yaml:8]: I think you should change the value of `retries` to `30` tto follow the template of the other services.

**Review by marianoleonardo — CHANGES_REQUESTED** [review]

File: roles/volumes/storageclass/tasks/main.yaml (hunk 1 above)
Comment (gabrielifg) [reply roles/volumes/storageclass/tasks/main.yaml]: Resolvido

**Review by ewerton-silva00 — APPROVED** [review]

**Review by danielapetito — APPROVED** [review]

**Review by marianoleonardo — APPROVED** [review]
Great Job!!! 👏🏼👏🏼👏🏼

**Pull request merged**